syslog-ng-helm-chart/templates/compress-job.yaml

```diff
@@ -0,0 +1,56 @@
+{{- if .Values.compressor.enabled }}
```

Comment:
We can shorter the scripts, if we eliminate the "enabled" flag, and test on **{{- if .Values.compressor }}** instead. This way the whole compressor section would be optional in the values file.